client/me/purchases/concierge-banner/index.jsx

```diff
@@ -119,7 +112,7 @@ class ConciergeBanner extends Component {
 					callToAction={ buttonText }
 					description={ description }
 					dismissPreferenceName={ `quick-start-banner-${ bannerType }` }
-					href={ `/me/quickstart/${ quickStartSite }` }
+					href={ `/me/quickstart/${ quickStartSiteSlug }` }
```

Comment:
To support more specific CSS, please specify a CSS class. (Location is only because it was close to an existing edit.)
```suggestion
					href={ `/me/quickstart/${ quickStartSiteSlug }` }
					className="concierge-banner"
```

Reply:
I added that class and a more specific rule.